frontend/expr_inferrer.py

```diff
@@ -382,8 +382,30 @@ def infer_func_call(node, context, solver):
 
 
 def infer_attribute(node, context, from_call, solver):
+    """Infer the type of attribute access
+    
+    Cases:
+        - Instance attribute access. ex:
+            class A:
+                def f(self):
+                    pass
+            A().f()
+        - Module access. ex:
+            import A
+            A.f()
+    """
+    # Check if it's a module access
     instance = infer(node.value, context, solver)
+
+    if isinstance(instance, Context):
```

Comment:
So I thought that, since we're handling module accesses in a static way, there are some cases that we don't support. And if that's the case, then there should be some code in here that is used in such a case that raises an error that says "you cannot use modules in this way". But there doesn't seem to be anything. At the VERY least there needs to be documentation that says what can go wrong and where.